lib/bulbs-elements/util/prep-ga-event.js

```diff
@@ -0,0 +1,18 @@
+export default function prepGaEvent (gaPrefix, gaId, dimensions = {}) {
+  ga('create', gaId, 'auto', { name: gaPrefix });
+
+  for (let key in dimensions) {
+    ga(`${gaPrefix}.set`, key, dimensions[key]);
+  }
+
+  // return function that sends google analytics event
+  return (eventCategory, eventAction, eventLabel, eventValue) => {
```

Comment:
We could make this a little bit more generalized.

return a function that can do more than just `send`

```js
return (method, ...args) => {
  return global.ga(`${gaPrefix}.${method}`, ...args);
}
```

Then you get a function that can be used anywhere the global `ga` function would be used, but it will get prefixed.